### ISSUE TITLE
Log extra fields not propagating

### ISSUE THREAD

**Issue by vitorguidi** [google/clusterfuzz#4200]
Extra log fields are not showing up in gcp, ie, [docker_image](https://github.com/google/clusterfuzz/blob/dc2f4acfe68b69e70f246e0adb1e687652ebc7c7/src/clusterfuzz/_internal/metrics/logs.py#L558)

This issue is meant to map out all missing fields and make sure they show up

**Comment by oliverchang** [comment]
I think this is likely due to https://github.com/google/clusterfuzz/pull/3422. These structured logs are only well supported if we use google-cloud-logging. We should hopefully be unblocked there now that the Python upgrade is complete?

**Comment by vitorguidi** [comment]
Yes, we are unblocked. We run logging 3.6 now ([ref](https://github.com/google/clusterfuzz/blob/master/src/Pipfile#L20))
I suggested this on the monitoring project

**Comment by oliverchang** [comment]
> Yes, we are unblocked. We run logging 3.6 now ([ref](https://github.com/google/clusterfuzz/blob/master/src/Pipfile#L20)) I suggested this on the monitoring project

Awesome, let's see if we can revive https://github.com/google/clusterfuzz/pull/3422 then

**Comment by karinacayres** [comment]
@oliverchang does the #3422 requires either delete the fluentd and also, adding the google.cloud.logging for bot logging?

**Comment by oliverchang** [comment]
> [@oliverchang](https://github.com/oliverchang) does the [#3422](https://github.com/google/clusterfuzz/pull/3422) requires either delete the fluentd and also, adding the google.cloud.logging for bot logging?

Yes both are needed for #3422.  #3422 should unblock this issue.
